__tests__/Auth0Client.test.ts

```diff
@@ -80,8 +80,23 @@ const loginWithPopup = loginWithPopupFn(mockWindow, mockFetch, fetchResponse);
 const checkSession = checkSessionFn(mockFetch, fetchResponse);
 
 describe('Auth0Client', () => {
+  const oldWindowLocation = window.location;
+
   beforeEach(() => {
-    mockWindow.location.assign = jest.fn();
+    // https://www.benmvp.com/blog/mocking-window-location-methods-jest-jsdom/
+    delete window.location;
+    window.location = Object.defineProperties(
+      {},
+      {
+        ...Object.getOwnPropertyDescriptors(oldWindowLocation),
+        assign: {
+          configurable: true,
+          value: jest.fn()
+        }
+      }
+    );
+    // --
```

Comment:
This **could** be pulled out into a common function for both `index.test.ts` and `Auth0Client.test.ts`, but we're about to move the relevant tests into one file anyway, so I anticipate this will only be needed in one of those files. We can refactor this out later.